.gitignore

```diff
@@ -1,27 +1 @@
-# Build
-lib/*
-
-###Node###
-
-# Logs
-logs
-*.log
-
-# Runtime data
-pids
-*.pid
-*.seed
-
-# Directory for instrumented libs generated by jscoverage/JSCover
-lib-cov
-
-# Coverage directory used by tools like istanbul
-coverage
-
-# Dependency directory
-# Commenting this out is preferred by some people, see
-# https://www.npmjs.org/doc/misc/npm-faq.html#should-i-check-my-node_modules-folder-into-git-
 node_modules
-
-# Users Environment Variables
-.lock-wscript
```

Comment:
Yup, I think we can add environnement / OS / editors specifics files in the .gitignore, it can prevent issues essaebtially from external contributors 👍